sprintRetrospective/week4.md

```diff
@@ -0,0 +1,66 @@
+- Include as many images as possible, e.g. console with no errors, terminal with no errors
+- Prototype - Get a user evaluation before starting the page
+- Finish off full pages. Don't leave things things unfinished for weeks
+- Communicate with the team as to where the data will be used. Who will need the data and in what format will it be needed. Check before you start doing it
+- Create spreadsheet of routes for both teams
+  - Inputs
+  - Outputs
+  - If something needs to be calculated, provide overview of what the function needs to do
+- Need to create a rule for branch/issue naming, everyone should follow this - "Epic - Issue". Help to link them back
+- Reject pull request if not linked to issue
+
+
+## **Stop - Things we need to stop doing:**
+- Don't rush pushing feature and signing off half-done features. 
+- Need to stop working on multiple features on a branch without committing, commit early and often.
+- Skimming pull requests without reading the full code. Try to understand the code and where it will be used. Has it been done already.
+
+
+## **Continue - Things we need to continue doing:**
+- Keep features separated so that we're not getting in each other's way. This worked well this week.
+- Overall good PRs so far. Keep it up comitting early and often.
+- Good and useful comments on the PRs. Keep this going.
+- Good communication as a whole. Discord is an extremely useful resoruce for keeping in contact.
+- Continue with frequency/length of meetings. We're avoiding unnecessary/overly long meetings.
+- More detailed reviews regarding efficiency of the code. E.g. stock data, don't return everything, just what is needed for the page.
+- If in doubt as to whether you should be in a meeting, ask the question.
+- Should aim to have a backup task to work on in case of slowness or depending on another coder is a suggestion.
+- Give a blocker maybe 45 mins - 1 hour of research on your own, but don't be afraid to ask the team in case they can point you in the right direction.
+
+
+## **Actions - Things we will be doing:**
+- Swap scrum lead - Warren in Joey out
+- Zenhub (everyone):
+  - Make sure to connect pull request to issue
+  - Update Timelines
+  - Add complexity (Score)
+- GitHub (everyone):
+  - Delete Branches when done
+  - Pull requests often, and for small components
+- Readme files for subfolders. Include anything needed to run locally (everyone)
+- Let people know that there's only a small review needed. (everyone)
+- Include as many images as possible, e.g. console with no errors, terminal with no errors (everyone)
+- Prototype - Get a user evaluation before starting the page (everyone)
+- Finish off full pages. Don't leave things things unfinished for weeks(everyone)
+- Communicate with the team as to where the data will be used. Who will need the data and in what format will it be needed. Check before you start doing it (everyone)
+- Create spreadsheet of routes for both teams (WHO?)
```

Comment:
For this point, I think @joeycor92 you said you'd make it this morning in the stand-up? Or did I misunderstand?